cli/tauri-bundler/src/bundle.rs

```diff
@@ -49,6 +52,34 @@ pub fn bundle_project(settings: Settings) -> crate::Result<Vec<PathBuf>> {
   settings.copy_resources(settings.project_out_directory())?;
   settings.copy_binaries(settings.project_out_directory())?;
 
+  #[cfg(windows)]
+  {
+    if let Ok(tauri_config) = get_tauri_config() {
+      if tauri_config.tauri.embedded_server.active {
+        let exempt_output = std::process::Command::new("CheckNetIsolation")
+          .args(&vec!["LoopbackExempt", "-s"])
+          .output()
+          .expect("failed to read LoopbackExempt -s");
+
+        if !exempt_output.status.success() {
+          panic!("Failed to execute CheckNetIsolation LoopbackExempt -s");
+        }
+
+        let output_str = String::from_utf8_lossy(&exempt_output.stdout).to_lowercase();
+        if !output_str.contains("win32webviewhost_cw5n1h2txyewy") {
+          println!("Running Loopback command");
+          Command::new("powershell")
```

Comment:
should be `std::process::Command::new("powershell")` and remove `.force_prompt(true)`